README.md

```diff
@@ -87,7 +87,7 @@ $ bin/yorkie
 
 Tests can be run by typing `make test`.
 
-*NOTE: `make test` includes integration tests that require a local MongoDB listen on port 27017. To start MongoDB, type `docker-compose up`.*
+*NOTE: `make test` includes integration tests that require a local MongoDB listen on port 27017. To start MongoDB, type `docker-compose -f docker/docker-compose up`.*
```

Comment:
Any reason not to mention `docker-compose-cluster-mode.yml`?

Our integration tests include ETCD and Prometheus.
In the actual `TestYorkie()` of `helper.go`, `Metrics` and `ETCD` option values ​​are added, and an instance is created with `yorkie.New()` function.  And at this time, `yorkie.New()` creates an actual related instance.

So I think it's more efficient to recommend the `docker-compose-cluster-mode.yml` file to the developer.

Let me know what you think.